tests/integration/incremental_unique_id_test/test_incremental_unique_id.py

```diff
@@ -0,0 +1,306 @@
+from tests.integration.base import DBTIntegrationTest, use_profile
```

Comment:
Incidentally, I looked into the circle ci logs. The reason failed is because it could not figure out how to cast a string to a date, which should in theory be solved by casting fields to explicit dates! Is there any way you think you could get your penultimate commit green? Would be nice to merge those tests. 

If possible, I'd love to see instead of duplicating all the code, we instantiate the classes with difference classes with a `profile` class variable (one for each database). Then you can enjoy more modular code!